dashboard/app/models/reference_guide.rb

```diff
@@ -0,0 +1,2 @@
+class ReferenceGuide < ApplicationRecord
+end
```

Comment:
please run the migration again, to add the schema info to this file and schema.rb